charts/renga/charts/gitlab/templates/_gitlab.rb.tpl

```diff
@@ -26,7 +26,7 @@ nginx['listen_https'] = false
 ### OmniAuth Settings
 ###! Docs: https://docs.gitlab.com/ce/integration/omniauth.html
 gitlab_rails['omniauth_enabled'] = true
-gitlab_rails['omniauth_auto_sign_in_with_provider'] = 'oauth2_generic'
+# gitlab_rails['omniauth_auto_sign_in_with_provider'] = 'oauth2_generic'
```

Comment:
any reason for this?

Reply:
this is just for now because without PR #220 there is no way to get into gitlab as admin